### DISCUSSION TITLE
How to make it easier to incorporate chat transcripts into minutes

### DISCUSSION THREAD

**Discussion by rjsparks** [ietf-tools/datatracker#3747] [Ideas]
Some chairs would like to attach the chat transcripts for a session into its minutes.

Right now, that requires cut-paste and some manual work to make the result legible in markdown.

Instead, the minutes upload form could offer to import from the chat logs while importing from the notes page.

Alternatively, we could provide a function that imported the chat logs into the notes page first, allowing the chairs to work with the content there before it becomes a version of the minutes.

**Comment by mcr** [comment]
Since minutes get reviewed and don't always get uploaded instantly (may take a few days), I guess that the import will be smart enough to know to grab the jabber logs for the day in which the meeting occurred.  And then there is complexity of WGs that had meetings on more than one day, but might post only a single set of minutes.

Conversely, a chair that uploads minutes in the two minutes before the end of the meeting might do so before discussion is actually finished...

Joel pointed out that he removed the join/leave messages from the logs, btw.

So I guess the best UI would be to auto-pick the jabber logs based upon the date, and then, offer to add more segments of logs, with some date picker.  Not sure how to deal with edits the chairs might want to make to the logs.  There could well be side discussions that we don't want to have in the minutes, maybe Code of Conduct violations we'd prefer not to amplify.

**Comment by rjsparks** [comment]
Archiving this as OBE. Modern datatracker covers this case via upload from the notes page.
